### ISSUE TITLE
function for restoring a backup through the UI

### ISSUE THREAD

**Issue by Shadesss** [Radarr/Radarr#3103]
Add a simple way for restoring a Radarr backup file through the UI similarly like we can make a backup now.

TL:DR tried moving from Radarr on Windows to Radarr on Docker and found restoring a backup was hard :)

----
Long-er version:
I had Radarr running on Windows and I have the backups. Now I want to move Radarr and some other things to run on Docker (on windows). Radarr image works just fine from the Docker Hub, but I can't figure out how to import/restore the backups and not break it.

If I enable the /config volume so to be able to replace the nzbdrone.db/config.xml files manually then Radarr just stops working for me. There doesn't seem to be an easy way I could import/restore the db without enabling the /config volume.
